src/librustc/middle/stability.rs

```diff
@@ -543,11 +543,6 @@ pub fn check_pat(tcx: &ty::ctxt, pat: &hir::Pat,
         // Foo(a, b, c)
         hir::PatEnum(_, Some(ref pat_fields)) => {
             for (field, struct_field) in pat_fields.iter().zip(&v.fields) {
-                // a .. pattern is fine, but anything positional is
```

Comment:
It's an erroneous comment, but it addresses a real issue, that of `Variant(..)` patterns.

I'm saying we move this comment up and have it talk of why `PatEnum(_, None)` is ignored.